refactor: :truck: move "why GitHub" into pre-workshop tasks (#240)
rostools/github-intro: sessions/introduction.qmd

<!-- TODO: I feel I am missing something here but can't remember -->

## A reminder: The big picture

::: {.callout-note collapse="true"}
## :teacher: Teacher note

> Verbally and briefly go over this section, to remind them about the

Reviewer: ```suggestion
Verbally and briefly go over this section, to remind the learners about the
```

reason and purpose of this workshop.
:::

As you read about in the pre-workshop tasks, GitHub is an increasingly
common tool that researchers use. You will likely encounter it in some
way as you do your work, either through collaborators or when reading
literature. As research becomes increasingly more dependent on
integrated digital tools and workflows, it is essential to be familiar
with these tools in order to be effective in present day research.

> This workshop aims to get you more familiar with GitHub and ready to use

Reviewer: ```suggestion
This workshop aims to get you more familiar with GitHub to help you get started with using
```

> it in your own work. We'll create a Git repository on GitHub with a

Reviewer: ```suggestion
it in your own work. During this workshop, we'll create a Git repository on GitHub with a
```
